### ISSUE TITLE
add version checks

### ISSUE THREAD

**Issue by guaka** [Trustroots/trustroots#624]
#618 spin-off

show warnings about versions
- [x] node (>=8 but 10 may not work correctly) — _Already done previosuly_
- [x] mongo (=3.4.x)  — Done in https://github.com/Trustroots/trustroots/pull/626

ideally these will show at
- [ ] npm install
- [x] npm start
- [ ] something else?

**Comment by guaka** [comment]
Did you also add a check for node or is that not needed?
My mac is on node 10.2.1 and it seems to work.

But if someone does `apt-get install nodejs` on Debian/Ubuntu they will end up with node 4.8 or so, which would definitely not work.

**Comment by simison** [comment]
>Did you also add a check for node or is that not needed?

https://github.com/Trustroots/trustroots/blob/a65f2daef904792b481e949abe74a7ab2693f5b6/package.json#L25

I forgot I added `npx check-node-version --package` earlier already. I did add a cap to not automatically support future Node.js versions before we actually confirm they work:
https://github.com/Trustroots/trustroots/blob/a65f2daef904792b481e949abe74a7ab2693f5b6/package.json#L12

Should be good now. 👍 

>My mac is on node 10.2.1 and it seems to work.

I think Node.js 10 works well enough although I'm not running it in production just yet. I might update soon though. 👍 

>But if someone does apt-get install nodejs on Debian/Ubuntu they will end up with node 4.8 or so, which would definitely not work.

Not sure if there's anything actionable for us here — feel free to improve docs. I felt like it added a bit too much noise to mention it in docs, especially now that we have version checking in place and docs specifically mention taking care of versions manually, too.
